### ISSUE TITLE
Example of this repo with react-native-router-flux

### ISSUE THREAD

**Issue by rgoldiez** [root-two/react-native-drawer#190]
@rt2zz - Do you have any examples of the proper way to use the drawer with `react-native-router-flux`?  To be honest, their docs are hard to follow and the "example" for using with with `react-native-drawer` is very incomplete.

**Comment by kevinvangelder** [comment]
@rgoldiez We're in the process of adding both packages to Ignite, so you might be able to gather some useful information from how I implemented them here: https://github.com/infinitered/ignite/pull/235

**Comment by rgoldiez** [comment]
Thanks @kevinvangelder ... this is very helpful!
